apps/music/style/music.css

```diff
@@ -1006,14 +1006,8 @@ html[dir="rtl"] li.list-item {
 
 li.list-item a {
   display: block;
-  color: #fff;
   text-decoration: none;
-  outline: none;
   margin: 0;
-  text-shadow: black 0.1rem 0.1rem;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
```

Comment:
@eeejay If I recalled correctly, we used to have `<a>` in a list element but we tried to reduce the children counts in the DOM tree, so `<a>` was removed in some commit, and this might be the styles we forget to clean.